stl/inc/chrono

```diff
@@ -6106,7 +6110,13 @@ template <class _Duration, class _CharT>
 struct formatter<_CHRONO local_time<_Duration>, _CharT> //
     : _Fill_tm_formatter<_CHRONO local_time<_Duration>, _CharT> {};
 
+template <class _CharT>
+struct formatter<_CHRONO weekday, _CharT> //

```

Comment:
what do these comments do?

Reply:
They force clang-format to wrap, which is nicer than disabling clang-format entirely. Otherwise, clang-format will put the base classes on the same line for some of the formatters (when the types are short enough), which obscures how the type is repeated.